scripts/setup_network_storage.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Google LLC
```

Comment:
I think this should be:
```
# Copyright (C) SchedMD LLC.
# Copyright 2024 Google LLC
```

AKA, both. This was guidance we were given when code was from SchedMD and we are altering.

You should probably put that in any file you touch which has a license.

Reply:
Updated.

Reply:
nit: You should probably put that in any file you touch which has a license.